esmvaltool/diag_scripts/validation.py

```diff
@@ -0,0 +1,252 @@
+"""
+Validation Diagnostic
+
+This diagnostic uses two datasets (control and experiment),
+applies operations on their data, and plots one against the other.
+It can optionally use a number of OBS, OBS4MIPS datasets.
+
+This diagnostic uses CMIP5 data; to switch to CMIP6 change _CMIP_TYPE
+"""
+
+import os
+import logging
+
+import numpy as np
+
+from esmvaltool.diag_scripts.shared import (group_metadata, run_diagnostic,
+                                            get_control_exper_obs,
+                                            apply_supermeans)
+from esmvaltool.preprocessor._area_pp import area_slice
```

Comment:
Use `from esmvaltool.preprocessor import extract_time, extract_season` instead of importing from private modules and put below third party module (iris, matplotlib) imports, as recommended by PEP8.